paasta_tools/list_kubernetes_service_instances.py

```diff
@@ -55,6 +56,15 @@ def parse_args():
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    parser.add_argument(
+        "-s",
```

Comment:
Would that conflict with -s --service?

Reply:
It wouldn't since -s is not used here. However, it might cause some confusion so I'll remove the short option.